tellmqtt/protocols/mandolyn.py

```diff
@@ -1,30 +1,48 @@
-# aka Clas Ohlson
+"""
+Protocol for mandolyn
+
+Tested models:
+ - ESIC WT450H (Clas Ohlson)
+"""
 
-import json
-from tellmqtt.common import protocoldata as pd
 import logging
 
+from tellmqtt.common import protocoldata as pd
+
 logger = logging.getLogger(__name__)
 
-class mandolyn:
+
+class Mandolyn:
```

Comment:
I would prefer the class name being kept as lower case to match the other protocols and the name used on the serial interface.
Otherwise it looks good

Reply:
No problem, updated PR. 😃